packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.Props.ts

```diff
@@ -107,4 +107,11 @@ export interface IChoiceGroupOption {
    */
   labelId?: string;
 
+  /**
+  * Optional input props that will be mixed into the input element, *before* other props are applied. This allows
+  * you to extend the input element with additional attributes, such as data-automation-id needed for automation.
+  * Note that if you provide, for example, "disabled" as well as "inputProps.disabled", the former will take
+  * precedence over the later.
+  */
+  inputProps?: React.HTMLAttributes<HTMLElement | HTMLInputElement>;
```

Comment:
I'm a bit confused; why do we need inputProps? You're extending the input element interface, so I would suspect that we would just extract the input props from the option itself.